recipes/xdesign/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "xdesign" %}
+{% set version = "v0.5.1" %}
```

I would remove the `v` from here:
```suggestion
{% set version = "0.5.1" %}
```

---

I assume this is because you don't want the `v` to be included in the package version string on conda?

---

Yeah. The version comparison is quite robust and can handle it but it's nice to keep strings clean where practical.